KMB/kmb_massload.py

```diff
@@ -311,33 +312,33 @@ def kmb_wrapper(idno):
 
 def load_list(filename='kmb_hitlist.json'):
     """Load json list."""
-    f = codecs.open(filename, 'r', 'utf-8')
-    data = json.load(f)
-    f.close()
-    return data
+    return common.open_and_read_file(filename, as_json=True)
 
 
 def output_blob(data, filename='kmb_data.json'):
-    """Dump data as jon blob."""
-    f = codecs.open(filename, 'w', 'utf-8')
-    f.write(json.dumps(data, indent=2, ensure_ascii=False))
-    print "%s created" % filename
+    """Dump data as json blob."""
+    common.open_and_write_file(filename, data, as_json=True)
+    pywikibot.output('{0} created'.format(filename))
 
 
 def run(start=None, end=None):
     """Get parsed data for whole kmb hitlist and store as json."""
+    log = common.LogFile('', 'kmb_massloading.log')
```

Comment:
Maybe define the name of the log file in the beginning of the script, so that one doesn't have to scroll all the way here to find/change it?